src-local/gle_utils.py

```
@@ -0,0 +1,206 @@
+"""
+Core utilities for solving the Generalized Lubrication Equations (GLE).
+
+This module contains the fundamental ODE system and solver functions
+without any dependencies on higher-level modules to avoid circular imports.
+"""
+
+import numpy as np
+from scipy.integrate import solve_bvp
+from functools import partial
+from typing import Tuple, Optional, Any
```

Ruff (0.11.9)

`typing.Tuple` imported but unused

Remove unused import

(F401)